src/mame/heathkit/intr_cntrl.cpp

```diff
@@ -38,7 +38,7 @@ void heath_intr_cntrl::device_reset()
 void heath_intr_cntrl::update_intr_line()
 {
 
-	m_irq_line(m_intr_lines == 0 ? CLEAR_LINE : ASSERT_LINE);
+	m_irq_line(m_intr_lines == 0 ? 0 : 1);
 }
```

Comment:
Can you please put parentheses around the condition in the ternary conditional operator like `(m_intr_lines == 0) ? 0 : 1`?  I know it works as-is, but it’s one of the things that’s more likely to be misinterpreted when someone’s debugging code while sleep-deprived.  Also, the blank line inside the function probably isn’t necessary.